src/main/java/taxi/service/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package taxi.service;
+
+import java.util.Optional;
+import taxi.dao.DriverDao;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Driver;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverDao driverDao;
```

Comment:
```suggestion
    private DriverService driverService;
```
![image](https://user-images.githubusercontent.com/81781479/187705514-a8a5ed2f-a253-4ab4-8343-8619f1ae839e.png)